src/test/cypress/README.md

```diff
@@ -38,7 +38,7 @@ For `cypress.json`:
 * `baseUrl`: The url pointing to the test environment here (make sure that there is no trailing slash)
```

Comment:
While you are at it you can also change the documentation link above in this file from https://artemis-platform.readthedocs.io/en/latest/ to https://docs.artemis.ase.in.tum.de/

Reply:
I don’t see a readthedocs link in this file, but only in `docs/README.md`. There it makes sense to keep, as only this link has the different versions. https://docs.artemis.ase.in.tum.de only has the latest one.

Reply:
Sorry, I wrote this to the wrong file. It was in the docs/README (line 63)